Header header
string oper_type
float32 answer 
float64 time_received
float64 time_answered
float64 process_time 
